tests/test_pdf.py

```diff
@@ -6,12 +6,21 @@
 path_tests = Path(__file__).parent
 
 
-def test_html_pdf(tmpdir):
+def test_pdfhtml(tmpdir):
     path_output = Path(tmpdir).absolute()
     path_template = path_tests.parent.joinpath("jupyter_book", "book_template")
-    cmd = f"jb build {path_template} --path-output {path_output} --build pdf_html"
+    cmd = f"jb build {path_template} --path-output {path_output} --builder pdfhtml"
     run(cmd.split(), check=True)
     path_html = path_output.joinpath("_build", "html")
     path_pdf = path_output.joinpath("_build", "pdf")
     assert path_html.joinpath("index.html").exists()
     assert path_pdf.joinpath("book.pdf").exists()
+
+
+def test_pdflatex(tmpdir):
+    path_output = Path(tmpdir).absolute()
+    path_template = path_tests.parent.joinpath("jupyter_book", "book_template")
+    cmd = f"jb build {path_template} --path-output {path_output} --builder pdflatex"
+    run(cmd.split(), check=True)
+    path_pdf = path_output.joinpath("_build", "latex")
+    assert path_pdf.joinpath("book.pdf").exists()
```

Comment:
we should add a TODO to make more comprehensive tests for latex pdf. I get the feeling that it will be quite finnicky, no? So we should at least do some sanity checks on the `tex` files. What do you think?

Reply:
yeah I think that is a good idea. will do.